internal/events/events.go

```diff
@@ -4,17 +4,20 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"io"
 	"os"
 	"os/signal"
 	"syscall"
+	"time"
 
 	"github.com/stellar/go/support/log"
 	"github.com/stellar/stellar-disbursement-platform-backend/internal/crashtracker"
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/utils"
 	"github.com/stellar/stellar-disbursement-platform-backend/stellar-multitenant/pkg/tenant"
 	"github.com/stretchr/testify/mock"
 )
 
+const MAX_BACKOFF_DURATION = 8
+
```

Comment:
👋  In Go, constants are usually CammelCased. This kind of constant is used in C and Cpp and shows up in Go usually when there's a Go lib that encapsulates a C code.

Reply:
My Python background rose from the ashes for a moment. 😅